cms/apps/pages/templatetags/pages.py

```diff
@@ -111,17 +118,21 @@ def meta_description(context, description=None):
     """
     if description is None:
         description = context.get("meta_description")
+
+    # TODO: Check in the context for objects for every templatetag like this
```

Comment:
We can do, i didn't do it here as it was "out of scope" for a Jinja change. I was gonna do it in a PR after, but i don't mind doing it in this PR.